lib/ma_crossover/events/data_managed_candles.js

```diff
@@ -56,45 +60,55 @@ const onDataManagedCandles = async (instance = {}, candles, meta) => {
         longIndicator.add(price)
       } else if (state.lastCandleLong.mts === lastCandle.mts) {
         longIndicator.update(price)
+      } else if (state.lastCandleLong.mts > lastCandle.mts) {
+        updateLastCandleLong = false // do nothing
```

Comment:
you don't want to log ignored candles, or better even log them, but flag as `ignored = 1` in the csv

its required for calcs

Reply:
added a new column `CANDLE_IGNORED`